### PR TITLE
alarm-clock-applet: remove

### PR DESCRIPTION
**Pull request by klemensn**
Last upstream update is from 2015, in 2020 it got abandoned by the
upstream maintainer.

The nixpkgs got a pending `-fno-common` fix, i.e. the package is
officially on life support.

This is unlikely to be revamped for GNOME3/GTK3, so better say goodbye.

See #39976.

@rasendubi

### CODE REVIEW

**Comment by klemensn** [comment]
Now also with correct GNOME2 deprecation issue number.

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -61,6 +61,7 @@ mapAliases ({
   aircrackng = throw "'aircrackng' has been renamed to/replaced by 'aircrack-ng'"; # Converted to throw 2022-02-22
   airtame = throw "airtame has been removed due to being unmaintained"; # Added 2022-01-19
   aleth = throw "aleth (previously packaged as cpp_ethereum) has been removed; abandoned upstream"; # Added 2020-11-30
+  alarm-clock-applet = throw "'alarm-clock-applet' has been abandoned upstream and depends on deprecated GNOME2/GTK2"; # Added 2022-06-16
```
Comment (rapenne-s) [inline pkgs/top-level/aliases.nix:64]: this line should be before `aleth` to keep the alphabetical order

**Review by rasendubi — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
> The nixpkgs got a pending `-fno-common` fix, i.e. the package is
> officially on life support.

That is nothing we should count because it is a common issues with all kinds of packages.

File: pkgs/top-level/aliases.nix (hunk 1 above)
Comment (SuperSandro2000) [inline pkgs/top-level/aliases.nix:64]:
```suggestion
  alarm-clock-applet = throw "'alarm-clock-applet' has been abandoned upstream and depends on deprecated GNOME2/GTK2"; # Added 2022-06-16
  aleth = throw "aleth (previously packaged as cpp_ethereum) has been removed; abandoned upstream"; # Added 2020-11-30
```

**Pull request merged**